src/ui/server/helpers/whole-number-validation/index.ts

```diff
@@ -5,21 +5,28 @@ import { stripCommas } from '../string';
 
 /**
  * validates if field is whole number and above 0 and handles commas in the input
- * returns validation error if is not a number, has a decimal place or special characters or is below 0
+ * if allowNegativeNumbers is set to true, then will return validation error if number below 0
+ * returns validation error if is not a number, has a decimal place or special characters
  * @param {RequestBody} responseBody
  * @param {object} errors
  * @param {string} errorMessage
  * @param {string} field fieldId of the field being checked
- * @param {number} minimum optional minimum value of field - default 0
+ * @param {Boolean} allowNegativeValue optional field set to false which allows for negative numbers below 0
```

Comment:
Please replace to `false as default, if false then allows for negative numbers below 0.`